docs/smart-contracts/getting-started/create-an-app.mdx

```diff
@@ -324,23 +325,30 @@ Current value: <strong id="current-value" aria-live="polite">???</strong><br />
 
 <script>
   import incrementor from "../contracts/soroban_increment_contract";
-
+  import { isAllowed, getUserInfo } from '@stellar/freighter-api';
   const button = document.querySelector("[data-increment]");
   const currentValue = document.querySelector("#current-value");
-
+  async function getPk() {
+    const { publicKey } = await getUserInfo();
+    return publicKey;
+  }
+  if (await isAllowed()) {
+    const publicKey = await getPk();
+    if (publicKey) incrementor.options.publicKey = publicKey;
+  }
   button.addEventListener("click", async () => {
     button.disabled = true;
     button.classList.add("loading");
     currentValue.innerHTML =
       currentValue.innerHTML +
       '<span class="visually-hidden"> – updating…</span>';
 
-    const tx = await incrementor.increment();
+    const tx = await incrementor.increment({timeoutInSeconds: 30});
```

Comment:
Do we need this? Should the default be increased from `10` so we don't need it anymore?

Reply:
It seems to be needed, which appears to be a bug - https://github.com/stellar/js-stellar-sdk/issues/950

Reply:
I took it out because the default will be changed to 60 for release.